test/new-e2e/tests/gpu/gpu_test.go

```diff
@@ -38,10 +40,49 @@ type gpuBaseSuite[Env any] struct {
 	caps                     suiteCapabilities
 	agentRestartsAtSuiteInit map[agentComponent]int
 	provisioner              provisioners.Provisioner
+	systemData               systemData
 }
 
 const vectorAddDockerImg = "ghcr.io/datadog/apps-cuda-basic"
 
+const (
+	gpuSystemUbuntu2204          systemName = "ubuntu2204"
+	gpuSystemUbuntu1804Driver430 systemName = "ubuntu1804-430"
+	gpuSystemUbuntu1804Driver510 systemName = "ubuntu1804-510"
+	defaultGpuSystem             systemName = gpuSystemUbuntu2204
+
+	cuda12DockerImage    = "669783387624.dkr.ecr.us-east-1.amazonaws.com/dockerhub/nvidia/cuda:12.6.3-base-ubuntu22.04"
+	pytorch19DockerImage = "669783387624.dkr.ecr.us-east-1.amazonaws.com/dockerhub/pytorch/pytorch:1.9.0-cuda10.2-cudnn7-runtime"
+)
+
+// gpuSystems is a map of AMIs for different Ubuntu versions
+var gpuSystems = map[systemName]systemData{
+	gpuSystemUbuntu2204: {
+		ami:                          "ami-03ee78da2beb5b622",
+		os:                           os.Ubuntu2204,
+		cudaSanityCheckImage:         cuda12DockerImage,
+		hasEcrCredentialsHelper:      false, // needs to be installed from the repos
+		hasAllNVMLCriticalAPIs:       true,  // 22.04 has all the critical APIs
+		supportsSystemProbeComponent: true,
+	},
+	gpuSystemUbuntu1804Driver430: {
+		ami:                          "ami-0cd4aa4912d788419",
+		cudaSanityCheckImage:         pytorch19DockerImage, // We don't have base CUDA 10 images from NVIDIA, so we use the PyTorch image
+		os:                           os.Ubuntu2004,        // We don't have explicit support for Ubuntu 18.04, but this descriptor is not super-strict
+		hasEcrCredentialsHelper:      true,                 // already installed in the AMI, as it's not present in the 18.04 repos
+		hasAllNVMLCriticalAPIs:       false,                // DeviceGetNumGpuCores is missing for this version of the driver,                                                                                                    // 430 driver has all the critical APIs
```

Comment:
```suggestion
		hasAllNVMLCriticalAPIs:       false,                // DeviceGetNumGpuCores is missing for this version of the driver,
		                                                                         // 430 driver has all the critical APIs
```
I think you wanted to do a newline rather than a long space